Toolchain/CMake/LLVMRuntimesConfig.cmake

```diff
@@ -1,9 +1,15 @@
 # This file specifies the options used for building the various LLVM runtime libraries
 
+# Note: We force the cmake module path for all dependent projects to include our custom directory
+# That has the Platform/SerenityOS.cmake definition
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${SERENITY_MODULE_PATH}" CACHE STRING "Modules for CMake")
+
 set(CMAKE_BUILD_TYPE Release CACHE STRING "")
 
 set(LLVM_ENABLE_RUNTIMES "libcxx;libcxxabi;libunwind" CACHE STRING "")
 
+set(CMAKE_SYSTEM_NAME SerenityOS CACHE STRING "")
+set(CMAKE_SYSTEM_PROCESSOR ${SERNITY_TOOLCHAIN_ARCH} CACHE STRING "")
```

Comment:
There's a typo in "`SERNITY_TOOLCHAIN_ARCH`".

Reply:
oh, in that case it's probably not needed at all :) I'll rebuild with that not set.